test/data/testGeoOpt.py

```diff
@@ -44,7 +44,8 @@ def testatomnos(self):
         # This will work only for numpy
         #self.assertEquals(self.data.atomnos.dtype.char, 'i')
 
-        atomnos_types = [numpy.issubdtype(atomno,int) for atomno in self.data.atomnos]
+        atomnos_types = [numpy.issubdtype(atomno, np.signedinteger)

```

Comment:
This is fine, but for the future generally it's better to separate no-op formatting changes from actual changes.